scripts/tests/qml_tests.sh

```diff
@@ -39,6 +63,11 @@ qmltest_compile || die
 print Y "Running the QML tests..."
 qmltest_run || die
 
+if [[ "$GRCOV_FILENAME" ]]; then
+  printn Y "Generating temp coverage file for lottie unit tests..."  
+  grcov -t lcov -o "$GRCOV_FILENAME" || die "Failure generating final qml lcov file"
```

Comment:
What does this command?
You must generate the grcov file using qmltest_grcov, right?